### DISCUSSION TITLE
Window tiled-box decorator

### DISCUSSION THREAD

**Discussion by YTN0** [mikke89/RmlUi#443] [General]
I'd there a doc or tutorial somewhere that shows the specific on how to chop up an image to use to frame a window / div?

I found the following, but it seems to be missing some details. 

https://mikke89.github.io/RmlUiDoc/pages/tutorials/window_template.html

It would be nice to have more details added to this for those of us more green on this topic.

Something visually explaining the pixel coordinates and cut sections would go a long way to making this easier to understand. 

Thanks!

**Comment by mikke89** [comment]
Hey.

In short, the process typically involves the following steps:

1. Start with creating a sprite sheet in your favorite image editor.
2. Then, define the individual sprites for that [sprite sheet in RCSS](https://mikke89.github.io/RmlUiDoc/pages/rcss/sprite_sheets.html). 
3. Finally, use those sprites in a [decorator](https://mikke89.github.io/RmlUiDoc/pages/rcss/decorators.html). For example, the [ninepatch decorator](https://mikke89.github.io/RmlUiDoc/pages/rcss/decorators/ninepatch.html) is a popular one that allows your element to be arbitrarily sized in both dimensions.

I recommend to take a look at the `demo` sample, as that has a little guide and demonstration of the different decorators. Maybe we should port this to the documentation. A PR would be very helpful.

There is also this nice GUI tool which you might find helpful: [RCSS Spritesheet Editor](https://github.com/svenvvv/rcss-sprite-ed), by @svenvvv.

Hopefully this is enough to point you in the right direction, but let me know if there are further details that can be clarified.
